configs/config_sandbox/esrf16_altmodels_study2/train-2dmodels-fpn-efficientnetb0.yaml

```diff
@@ -0,0 +1,19 @@
+train_config:
+  model_id_prefix: 'segmentation-model-fpn-efficientnetb0-study2'
+  dataset_id: 'dataset-esrf16_study2_CropCase5_TrnValCaseB'
+  segmentation_model:
+    model_name: 'FPN'
+    model_parameters:
+      backbone_name: 'efficientnetb0'
+      encoder_weights: Null   # Null (random initialization) or 'imagenet' (pre-training on ImageNet, only RGB images)
+      input_shape: [Null, Null, 1]
+      activation: 'sigmoid'
+  loss: 'cross_entropy'
```

Comment:
this should say `'binary_cross_entropy'` (make sure `models.py reflects this choice also).

Reply:
done on the `models.py` side

Reply:
done on configs too